tests/unit/steps/test_generate_lz4.py

```diff
@@ -0,0 +1,45 @@
+import os
+import time
+import unittest
+
+from tests.unit.unittest_helpers import relative_file_path
+
+from idseq_dag.steps.generate_lz4 import PipelineStepGenerateLZ4
+
+INPUT_FILE = relative_file_path(__file__, 'dummy testfile.txt')
+OUTPUT_FILE = INPUT_FILE + '.lz4'
+INPUT_FILE_GZIP = INPUT_FILE + '.gz'
+OUTPUT_FILE_GZIP = INPUT_FILE_GZIP + '.lz4'
+
+class TestPipelineStepGenerateLZ4(unittest.TestCase):
+
+    def setUp(self):
+        self.step = PipelineStepGenerateLZ4(
+            name='test_generate_lz4',
+            input_files=[[INPUT_FILE, INPUT_FILE, INPUT_FILE_GZIP]],
+            output_files=[OUTPUT_FILE, OUTPUT_FILE, OUTPUT_FILE_GZIP],
+            output_dir_local='',
```

Comment:
I noticed you are running your command using a dummy test file.
I have a suggestion: should we use something like `/tmp/test_pipeline_step_generate_lz4` as output folder?
I usually avoid writing files during tests to the local folder because it has some side effects, like generating eventual garbage during commits, raising errors if you are running a docker image with local folder in read-only mode, or even triggering new test execution if you are watching source folders for file changes.